geode-docs/reference/topics/gemfire_properties.html.md.erb

```diff
@@ -488,26 +488,31 @@ off-heap-memory-size=120g</code></pre></td>
 </tr>
 <tr>
 <td>redis-bind-address</td>
-<td>Specifies the bind address used by Redis API for <%=vars.product_name%>. If "" or no bind address
-specified, the default will be requested from the operating system.</td>
+<td>Specifies the address on which the Redis API for <%=vars.product_name%> is listening. If set to the empty string or this property is not
+specified, localhost is requested from the operating system.</td>
 <td>S</td>
-<td>""</td>
+<td>`""`</td>
```

Comment:
This renders wrong.  Markdown and HTML are not playing together. Please try

<td><code>""</code></td>

Reply:
My attempt to give you the exact keystrokes to put in here got rendered. Trying again.  
```
<td><code>""</code></td>
```

Reply:
Updated it! It seems to look ok now.